### PR TITLE
[DFMD-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Mistura-Yusuf**
I just added page navigation to my pages. Please review

### CODE REVIEW

File: submissions/misturaYusuf/html-forms/contact-form.html [hunk 1]
```diff
@@ -4,6 +4,15 @@
 	<title>Contact Form</title>
 </head>
 <body>
+	<nav>
+		<h1><a href="file:///C:/Users/HP/Desktop/Dufuna-Fem/submissions/misturaYusuf/html/product-page.html">MISSTYB'S GLAMOROUS TOUCH</a></h1>
```
Comment (Ireti203) [inline submissions/misturaYusuf/html-forms/contact-form.html:8]: This file path can only work on your computer. I don't have this exact directory. You can update the link to '../html/product-page.html'

File: submissions/misturaYusuf/html-forms/contact-form.html [hunk 2]
```diff
@@ -4,6 +4,15 @@
...
+		<ul>
+			<li><b><a href="#">Our Story</a></b></li>
+			<li><b><a href="file:///C:/Users/HP/Desktop/Dufuna-Fem/submissions/misturaYusuf/html/product-page.html">Product</a></li></b></li>
```
Comment (Ireti203) [inline submissions/misturaYusuf/html-forms/contact-form.html:11]: Same here

Comment (Ireti203) [inline submissions/misturaYusuf/html-forms/contact-form.html:11]: You should also add the link to the contact-form

File: submissions/misturaYusuf/html/product-page.html [hunk 3]
```diff
@@ -6,12 +6,12 @@
 </head>
 <body>
 	<nav>
-		<h1><a href="product page.html">MISSTYB'S GLAMOROUS TOUCH</a></h1>
+		<h1><a href="product-page.html">MISSTYB'S GLAMOROUS TOUCH</a></h1>
```
Comment (Ireti203) [inline submissions/misturaYusuf/html/product-page.html:9]: Add the link to the contact-form

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by Ireti203** [comment]
Ensure it navigates on your computer.

**Comment by Mistura-Yusuf** [comment]
The links navigates to the respective pages now. Please review. Thank you

**Comment by Mistura-Yusuf** [comment]
Please review my PR

**Review by adaugochi — APPROVED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**